contracts/examples/hello-world/tests/hello-world.ts

```diff
@@ -3,7 +3,7 @@ import * as fs from 'fs';
 import { Program, BN } from '@project-serum/anchor';
 import { HelloWorld } from '../target/types/hello_world';
 
-const CHAINLINK_PROGRAM_ID = "A7Jh2nb1hZHwqEofm4N8SXbKTj82rx7KUfjParQXUyMQ";
+const CHAINLINK_PROGRAM_ID = "CaH12fwNTKJAG8PxEvo9R96Zc2j8qNHZaFj8ZW49yZNT";
```

Comment:
Already defined in `contracts/examples/hello-world/client.js`. Let's use it from there.

Reply:
The test file doesn't use the client.js yet. I'm going to add a proper ta client in a follow-up that will clean up both files